app/src/main/java/org/mozilla/fenix/library/history/HistoryFragment.kt

```diff
@@ -106,4 +123,9 @@ class HistoryFragment : Fragment(), CoroutineScope {
             else -> super.onOptionsItemSelected(item)
         }
     }
+
+    override fun onBackPressed(): Boolean {
+        if ((historyComponent.uiView as HistoryUIView).onBackPressed()) { return true }
```

Comment:
It seems this could be simplified to `override fun onBackPressed(): Boolean = (historyComponent.uiView as HistoryUIView).onBackpressed()`

Reply:
:man_facepalming: good catch